utils/tools.py

```diff
@@ -0,0 +1,47 @@
+from functools import wraps
+from time import time
+
+palette = (2 ** 11 - 1, 2 ** 15 - 1, 2 ** 20 - 1)
+
+
+def compute_color_for_labels(label):
+    """
+    Simple function that adds fixed color depending on the class
+    """
+    color = [int((p * (label ** 2 - label + 1)) % 255) for p in palette]
+    return tuple(color)
+
+
+def is_video(ext: str):
+    """
+    Check if a video input is in supported foramts.
+    Args:
+        ext:
+
+    Returns:
+
+    """
+    allowed_exts = ('.mp4', '.webm', '.ogg', '.avi')
+    return any((ext.endswith(x) for x in allowed_exts))
+
+
+def tik_tok(func):
```

Comment:
Now you can use this function as decorator for detection function in yolov3_deepsort.py .